client/src/app/site/projector/components/projector-detail/projector-detail.component.html

```diff
@@ -231,3 +236,58 @@ <h2>{{ projector?.name | translate }}</h2>
         </div>
     </div>
 </div>
+
+<!-- Countdown dialog -->
+<ng-template #countdownDialog>
+    <h1 mat-dialog-title>
+        <span translate>Countdown</span>
+    </h1>
+
+    <form (ngSubmit)="submitCountdown(editCountdown)" #countdownForm="ngForm">
+        <div mat-dialog-content>
+            <!-- Title field -->
+            <mat-form-field>
+                <input
+                    matInput
+                    osAutofocus
+                    placeholder="{{ 'Countdown title' | translate }}"
+                    [(ngModel)]="editCountdown.title"
+                    name="countdown-title"
+                    required
+                />
+            </mat-form-field>
+
+            <!-- Description field -->
+            <mat-form-field>
+                <input
+                    matInput
+                    placeholder="{{ 'Countdown description' | translate }}"
+                    [(ngModel)]="editCountdown.description"
+                    name="countdown-description"
+                />
+            </mat-form-field>
+
+            <!-- Time field -->
```

Comment:
Please use the durationTImeService as in the old component. Maybe use a form instead of `ngModel`